src/streamlink/plugins/crunchyroll.py

```diff
@@ -39,6 +39,9 @@ def parse_timestamp(ts):
 })
 _media_schema = validate.Schema(
     {
+        "name": validate.text,
+        "series_name": validate.text,
+        "media_type": validate.text,
```

Comment:
Does the API response always include these properties when you add the additional fields to the request (down below)? Are they always strings or can there be null values? I'm asking, because metadata should be optional if possible, to not break anything.